apps/examples/wifi_test/wifi_test.c

```diff
@@ -248,7 +249,9 @@ int main(int argc, FAR char *argv[])
 int wifi_test_main(int argc, char *argv[])
 #endif
 {
-
+#if CONFIG_TASH
```

Comment:
This should be "#ifdef CONFIG_TASH", not "#if CONFIG_TASH". Otherwise, compilation error will occur when CONFIG_TASH is not set.